client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -179,15 +205,8 @@ export default React.createClass( {
 		}
 	},
 	render: function() {
-		let pipe = this.getPipe();
-
-		for ( let renderer of pipe ) {
-			let result = renderer();
-			if ( result ) {
-				return result;
-			}
-		}
-		return null;
+		const notices = this.getPipe().map( renderer => renderer() ).filter( notice => notice );
```

Comment:
Because you can have 2 or 3 or 5 critical messages. If it's too ugly to display 5 critical notices one below the other, we should improve this component instead of hiding 4 of them.